tests/drivers/spi/spi_loopback/boards/nucleo_wb55rg.overlay

```diff
@@ -12,10 +12,12 @@
 
 &dma1 {
 	status = "okay";
+	dma-offset = <0>;
```

Comment:
This is soc dependent, should be moved to `dts/arm/st`

Reply:
moved to soc, to be applied on each stm32 that enables its dmamux